portal/models/i18n_utils.py

```diff
@@ -302,3 +308,177 @@ def compile_pos():
             "Saved MO file: %s",
             os.path.relpath(mo_filepath, current_app.root_path),
         )
+
+
+def upsert_to_template_file():
+    db_translatables = {}
+    db_translatables.update(get_db_strings())
```

Comment:
what's the value of creating an empty dict just to update it?  why not:
`db_translatables = get_db_strings()`